README

```diff
@@ -2,6 +2,8 @@
 
 We are working on a vendor Kernel for the TP-Link M7350 V3 here. We grabed the Sources from the TP-Link GPL Section and started patching to get it building with a modern Toolchain.
 
+To build this kernel, the python command needs to point to a python3 environment.
```

Comment:
... the `python` command needs to be Python 3

just that; "environment" may be confusing